isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/Chandrayaan1M3Camera.truth

```diff
@@ -32,8 +32,8 @@ DeltaLine = 0.000000000
 For center pixel position ...
 Latitude OK
 Longitude OK
-RightAscension = 242.182569911
-Declination = -84.617919659
```

Comment:
So are these values changing due to updated spice data?

Reply:
I believe so, yes.

Reply:
This is not a trivial change, but also not out of the ordinary when updating spice spk and/or ck, and/or dem